README.md

```diff
@@ -40,15 +40,15 @@ results_df = df.prql.query("select [age,name,occupation] | filter age > 21")
 ```python
 In [1]: %load_ext pyprql.magic
 In [2]: %prql postgresql://user:password@localhost:5432/database
-In [3]: %%prql
+In [3]: %config PrqlMagic.target="sql.postgres"
```

Comment:
Overall this is great, but I think we should try and keep the first example extremely simple; removing whatever we can, because the budget for people to spend time understand new things is quite small.

So I think we should remove this. If we want, we could add it to another example.

(And possibly we should be doing this automatically based on the current connection, but that's a different piece of work)

Reply:
I agree.

Documentation needs to be improved in the future.